flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/MaterializedTableStatementITCase.java

```diff
@@ -167,28 +169,42 @@ void before() throws Exception {
 
         fileSystemCatalogPath = fileCatalogPath.toString();
         fileSystemCatalogName = TEST_CATALOG_PREFIX + randomStr;
+        // initialize session handle, create test-filesystem catalog and register it to catalog
+        // store
+        sessionHandle = initializeSession();
     }
 
     @AfterEach
-    void after(@InjectClusterClient RestClusterClient<?> restClusterClient) throws Exception {
-        // cancel all running jobs for releasing mini cluster resources
-        for (JobStatusMessage j : restClusterClient.listJobs().get()) {
-            // cancel all continuous refresh jobs
-            if (j.getJobName().endsWith("continuous_refresh_job")
-                    && (j.getJobState() == JobStatus.RUNNING
-                            || j.getJobState() == JobStatus.CREATED)) {
-                restClusterClient.cancel(j.getJobId()).get();
+    void after() throws Exception {
+        Set<TableInfo> tableInfos =
+                service.listTables(
+                        sessionHandle,
+                        fileSystemCatalogName,
+                        TEST_DEFAULT_DATABASE,
+                        Collections.singleton(CatalogBaseTable.TableKind.TABLE));
```

Comment:
I think the correct table kind should be `MATERIALIZED TABLE`, we should extend the function of `listTables` method, it should support return materialized table. and the `listTables` method should return the correct table kind, table is table, materialized table is materialized table, these are two things.